src/Admin/Services/AccessControlService.cs

```diff
@@ -17,6 +21,18 @@ public class AccessControlService : IAccessControlService
         _globalSettings = globalSettings;
     }
 
+    public bool UserHasPermission(Permission permission)
+    {
+        if (_globalSettings.SelfHosted)
+            return true;
```

Comment:
we try to always [use curly braces for control blocks](https://contributing.bitwarden.com/contributing/code-style/csharp#control-blocks)